resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -1,38 +1,60 @@
 #!/usr/bin/env python3
-# pylint: disable=missing-docstring,invalid-name,import-error
 """Create IGV session for archive."""
 
 import argparse
-from lxml import etree
 import os
 
-parser = argparse.ArgumentParser(description='Create igv session')
+from lxml import etree
+
+parser = argparse.ArgumentParser(description="Create igv session.")
 
-parser.add_argument("-f", "--files", nargs='+', required=True, help="List of files")
-parser.add_argument("-g", "--genome", type=str, required=True, help="Genome")
+parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV")
 
 args = parser.parse_args()
 
-Global = etree.Element(
-    'Global',
-    genome=args.genome,
-    version='8'
-)
 
-Resources = etree.SubElement(Global, 'Resources')
+def make_xml_tree(input_file):
+    """Make xml tree for IGV session."""
+    global_ = etree.Element(  # pylint: disable=no-member
+        'Global',
+        version='3',
+    )
+
+    resources = etree.SubElement(global_, 'Resources')  # pylint: disable=no-member
+
+    with open(input_file, 'r') as tfile:
+        for line in tfile:
+            element_name = os.path.basename(line).rstrip()
+            element_path = os.path.join('..', line.rstrip())
+            etree.SubElement(  # pylint: disable=no-member
+                resources,
+                'Resource',
+                name=element_name,
+                path=element_path,
+            )
+
+    doc = etree.tostring(  # pylint: disable=no-member
+        global_,
+        pretty_print=True,
+        xml_declaration=True,
+        encoding='UTF-8',
+    )
+    return doc
+
 
-for element in args.files:
-    element_name = os.path.basename(element)
-    element_dir = os.path.dirname(os.path.dirname(element))
-    element_path = os.path.join('..', element)
+def write_xml_file(input_file, doc):
+    """Compose a name and write output file."""
+    name = input_file
+    name = name.replace('temp_igv.txt', 'igv.xml')
+    with open(name, 'wb') as f:
```

Comment:
`str.replace()` does not make in-place modifications, but makes a copy and modifies it. Check [the docs](https://docs.python.org/3/library/stdtypes.html#str.replace).You can delete upper two lines and just write `with open(input_file.replace('temp_igv.txt', 'igv.xml'), 'wb') as f:`